sessions/css-positioning/demo-end/index.html

```diff
@@ -6,215 +6,119 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>HTML & CSS</title>
+    <title>CSS Positioning</title>
     <link rel="stylesheet" href="./css/styles.css" />
   </head>
   <body>
     <header class="header">
-      <h1 class="header__heading">CSS positioning</h1>
+      <h1 class="header__heading">
+        Sweeties<span class="plate" role="img" aria-label="A plate">🍽</span>
+      </h1>
     </header>
-    <main>
-      <section class="disclaimer">
-        <h2 class="disclaimer__title">disclaimer</h2>
-        <div class="disclaimer__icon">!</div>
-        <p class="disclaimer__text">
-          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Alias vitae
-          libero harum! Odit consequatur veritatis, hic qui, facere distincLorem
-          ipsum dolor sit, veritatis, hic qui, facere distincorem ipsum dolor
-          sit, amet consectetur adipisicing elit. Alias vitae libero harum! Odit
-          consequatur veritatis, hic qui, facere distinc
+
+    <main class="main">
+      <section class="main__section">
+        <p class="main__text">
+          From hard candies to gummies, chocolates to caramels, we've got
+          something for everyone to satisfy their sweet tooth!
+        </p>
+        <span class="donut" role="img" aria-label="A chocolate donut">🍩</span>
+        <span class="donut-text">Delicious!</span>
+        <p class="main__text">
+          First, our hard candies come in a variety of flavors, including
+          classic favorites like cherry, lemon, and lime, as well as unique
+          blends like watermelon-mint and strawberry-basil. They're perfect for
+          enjoying slowly, letting the flavor melt and linger in your mouth.
+        </p>
+        <p class="main__text">
+          For those who prefer a chewier candy, our gummies are a must-try.
+          We've got everything from classic gummy bears to sour worms and fruit
+          slices, with flavors like grapefruit, peach, and raspberry. Our
+          gummies are soft and chewy, bursting with fruity goodness.
+        </p>
+        <p class="main__text">
+          For the chocolate lovers out there, Sweeties has you covered with our
+          delicious chocolate creations. From rich, creamy truffles to
+          chocolate-covered pretzels, there's a treat to suit every taste. We
+          use only the finest ingredients, including premium cocoa and fresh
+          nuts, to create our chocolates.
+        </p>
+        <span class="separator" role="img" aria-label="A piece of hard candy"
+          >🍬</span
+        >
+        <p class="main__text">
+          Our caramels are some of the best around. We've got classic flavors
+          like vanilla and chocolate, as well as unique twists like sea salt and
+          honey. Our caramels are soft and buttery, with just the right amount
+          of sweetness.
+        </p>
+        <p class="main__text info">
```

Comment:
```suggestion
        <p class="content__text content__info">
```

Reply:
I'm not sure if that's how BEM works. It could be argued that it should be `content__text content__text--info`, because it's a variant of `content__text`, but that's really up in the air imho.

Reply:
You are right, I think content__text--info is the way to go. Imo the point of BEM is to be able to make sense of what and where the defined class is used for, just by the help of the class name. Just "info" doesn't really do that, because if you just see this class it could be located and used anywhere. In this case I think that is not the purpose of this info.